_commands/basics/mv.md

```diff
@@ -0,0 +1,67 @@
+---
+---
+
+mv
+--
+
+`mv` is used to move a source file or directory into a destination directory. It is also commonly used to rename files. 
+
+~~~ bash
+$ mv source desination
+~~~
+
+<!--more-->
+
+### Useful Options / Examples
+
+#### `mv source destination`
+~~~ bash
+$ ls 
+file1 file2 file3 directory1
+$ mv file1 directory1 
+$ ls 
+file2 file3 directory1 
+$ ls ../directory1
+file1
```

Comment:
You can replace the last two `ls`'s with `tree`, which is a nice way to show off that command too and give slightly clearer output:

```bash
$ tree .
.
├── dir1
│   └── file1
├── file2
└── file3
```